AubreyV/asm_system
# Added Deactivate Confirmation UI

## Description / Purpose (checkpoints)
This PR fixes issue https://github.com/AubreyV/asm_system/issues/12.

## Type of change
 - New feature (non-breaking change which adds functionality)

## How To Test This PR
 1. Go to URL below 
 2. Click trash icon
 3. Check if there's a confirmation modal

 **Commands**
  - `npm run dev`

 **Routes/URL**
  - `/admin/users`
